Homework2510/Homework2510/Controllers/DataFileController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Homework2510.Models;
+
+namespace Homework2510.Controllers
+{
+    [Route("DataFile")]
+    [ApiController]
+    public class DataFileController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+        private readonly IWorkWithFile _iworkWithFile;
+        private readonly string _name;
+        public DataFileController(IConfiguration config, IWorkWithFile iWorkWithFile)
+        {
+
+            _config = config;
+            _name = _config.GetValue<string>("DataFile");
+            _iworkWithFile = iWorkWithFile;
+        }
+        [HttpGet("Get_Name")]
```

Comment:
endpoint uri should contain only lower case and '-' for work separation
rename this to 'get-name' and all the rest of endpoints to match this convention